src/scripts/main.js

```diff
@@ -1,3 +1,399 @@
+        
+        <option value="San Francisco">San Francisco</option>
+      </select>
+    </label>
+
+    <label>
+      Age:
+      <input
+        name="age"
+        type="number"
+        data-qa="age"
+      >
+      </input>
+    </label>
+
+    <label>
+      Salary:
+      <input
+        name="salary"
+        type="number"
+        data-qa="salary"
+      >
+      </input>
+    </label>
+
+    <button
+      type="submit"
+    >
+      Save to table
+    </button>
+  </form>
+`);
+
+function callNotification(title, type, text) {
+  const notification = document.createElement('div');
+
+  notification.dataset.qa = 'notification';
+
+  notification.classList.add('notification', type);
+
+  notification.innerHTML = `
+    <h2 class="title">${title}</h2>
+    ${text}
+  `;
+
+  body.append(notification);
+
+  setTimeout(() => {
+    notification.remove();
+  }, 3000);
+};
+
+const form = document.querySelector('form');
+
+form.addEventListener('submit', () => {
+  event.preventDefault();
+
+  const data = new FormData(form);
+
+  const formValues = Object.fromEntries(data.entries());
+
+  const money = +(formValues.salary);
+
+  const newRow = `
+    <tr>
+      <td>
+        ${formValues.name}
+      </td>
+
+      <td>
+        ${formValues.position}
+      </td>
+
+      <td>
+        ${formValues.office}
+      </td>
+
+      <td>
+        ${formValues.age}
+      </td>
+
+      <td>
+        $${money.toLocaleString('en-US')}
+      </td>
+    </tr>
+  `;
+
+  if (formValues.name.length < 4
+    || isNaN(formValues.name) === false
+    || formValues.age < 18
+    || formValues.age > 90
+    || formValues.position.length === 0
+    || isNaN(formValues.position) === false
+    || formValues.salary === 0
+  ) {
+    callNotification('Error', 'error', `
+      <p>You need to fill all fields</p>
+      <p>Length of name must be not less than 4 letters</p>
+      <p>You don't need to use numbers in Name and Position fields</p>
+      <p>Age must be not less than 18 and not more than 90 years old</p>
+    `);
+  } else {
+    callNotification('Success', 'success', `
+      <p>Information added to the table</p>
+    `);
+
+    bodyOfTable.insertAdjacentHTML('beforeend', newRow);
+  }
+});
+
+bodyOfTable.addEventListener('dblclick', () => {
+  const cellInitialValue = event.target.innerHTML;
+  const row = event.target.parentElement;
+  const position = event.target.cellIndex;
+  const currentCell = row.children[position];
+
+  if (position === 2) {
+    event.target.innerHTML = `
+      <select
+        name="office"
+        data-qa="office"
+        class="cell-input"
+      >
+        <option value="Tokyo">Tokyo</option>
+
+        <option value="Singapore">Singapore</option>
+
+        <option value="London">London</option>
+
+        <option value="New York">New York</option>
+
+        <option value="Edinburgh">Edinburgh</option>
+        
+        <option value="San Francisco">San Francisco</option>
+      </select>
```

Comment:
don't duplicate this

Reply:
I need to use differect classe for the select in form and in the cell so i reused option list only